androidClient/app/src/main/res/layout/content_about_us.xml

```diff
@@ -58,30 +57,73 @@
         android:id="@+id/content_about_us_fb_imgbtn"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:layout_margin="4dp"
-        android:layout_marginTop="8dp"
+        android:layout_marginLeft="112dp"
+        android:layout_marginTop="4dp"
         android:background="@color/mac_color"
         android:textColor="@color/white"
         app:layout_constraintHorizontal_chainStyle="packed"
         app:layout_constraintLeft_toLeftOf="parent"
         app:layout_constraintRight_toLeftOf="@+id/content_about_us_google_play_imgbtn"
+        app:layout_constraintLeft_toRightOf="@id/content_about_us_github_imgbtn"
         app:layout_constraintTop_toBottomOf="@+id/content_about_us_mac_img"
         app:srcCompat="@drawable/ic_facebook_white" />
 
     <ImageButton
         android:id="@+id/content_about_us_google_play_imgbtn"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:layout_margin="4dp"
-        android:layout_marginTop="8dp"
+        android:layout_marginRight="210dp"
+        android:layout_marginTop="4dp"
         android:background="@color/mac_color"
         android:textColor="@color/white"
         app:layout_constraintHorizontal_chainStyle="packed"
         app:layout_constraintLeft_toRightOf="@+id/content_about_us_fb_imgbtn"
         app:layout_constraintRight_toRightOf="parent"
+        app:layout_constraintLeft_toLeftOf="@id/content_about_us_linkedin_imgbtn"
         app:layout_constraintTop_toBottomOf="@+id/content_about_us_mac_img"
         app:srcCompat="@drawable/ic_google_play" />
 
+    <ImageButton
+        android:id="@+id/content_about_us_github_imgbtn"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="4dp"
+        android:layout_marginLeft="80dp"
+        android:background="@color/mac_color"
+        android:textColor="@color/white"
+        app:srcCompat="@drawable/ic_github_circle"
+        app:layout_constraintHorizontal_chainStyle="packed"
```

Comment:
What does this do?

Reply:
this is github icon

Reply:
the `app:layout_constraintHorizontal_chainStyle="packed"` thing